src/client/pages/OfferNew/Introduction/Sidebar/index.tsx

```diff
@@ -18,13 +18,16 @@ import {
 } from 'pages/OfferNew/Introduction/Sidebar/utils'
 import { OfferData } from 'pages/OfferNew/types'
 import { useTextKeys } from 'utils/textKeys'
+import { Badge } from 'components/Badge/Badge'
 import { Price } from '../../components'
 import { insuranceTypeTextKeys, isBundle } from '../../utils'
 import { DetailsModal } from './DetailsModal'
 import { DiscountCodeModal } from './DiscountCodeModal'
 import { StartDate } from './StartDate'
 import { StickyBottomSidebar } from './StickyBottomSidebar'
 
+export const SIDEBAR_BP = '@media (min-width: 1020px)'
```

Comment:
To me `BP` indicates the breakpoint value - like just the number or the number + `px`. Maybe call it something like `SIDEBAR_DESKTOP_MEDIA_QUERY`? Then you also know that it's a media query for specifically desktop 😀

Reply:
I wanted to align this with the the naming in `market-web` where we have `const LAPTOP_BP_UP = '@media (min-width: 1024px)'` for example. I can change the name to desktop or tablet or something similar if we think it makes more sense 👍

Reply:
Ah, OK, I see. Yes that would be amazing! 🙏